tikv-configuration-file.md

```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
 + Default value: `MIN(CPU * 0.75, 32)`.
 + Minimum value: `1`
 
+## cdc <span class="version-mark">New in v4.0.5</span>
```

Comment:
This version mark should be removed from master

Reply:
```suggestion
## cdc
```